entity-framework/core/modeling/relationships.md

```diff
@@ -92,13 +92,13 @@ See [Cascade Delete](../saving/cascade-delete.md) for more details about the dif
 
 ## Data Annotations
 
-There are two data annotations that can be used to configure relationships, `[ForeignKey]` and `[InverseProperty]`.
+There are two data annotations that can be used to configure relationships, `[ForeignKey]` and `[InverseProperty]`. These are available in the `System.ComponentModel.DataAnnotations` assembly.
```

Comment:
@divega the original bug specifically mentioned that they didn't know which assembly to use. Still feel the same?

Reply:
Good point, that is what he asked, but I wonder if he did it out of custom. I don’t know how the assembly reference would have help him, since that is not how you acquire it when you use EF Core. Now that I think about it from that angle, the package doesn’t help much either because it gets added automatically, unless you want to build the model into a project that doesn’t reference EF Core. Maybe the namespace is all he needed?

On the other hand, we should consider pointing to the API browser for matters like this, but unfortunately when you point to something like https://docs.microsoft.com/dotnet/api/system.componentmodel.dataannotations.requiredattribute, it defaults to .NET Framework and also shows the assembly that isn’t generally how you should acquire it or look for it.

Reply:
@lajones @divega I think it's the namespace people really want; If you're not using VS/Rider, then that can be hard to find. (Both VS and Rider will prompt you to insert the `using`, so for those IDE's even the namespace is additional clutter in the docs.)

Like @divega said, the way you acquire the assembly is by installing the EF package, which brings in the Data Annotations package automatically. So it's hard to see how knowing anything about the assembly/package would help _unless_ it is as a way to find the namespace.

Reply:
Ok, then I am convinced we just need to specify the namespace when it isn’t obvious.

Reply:
Updated to refer to the namespace.